src/main/java/org/opentripplanner/transit/api/request/TripOnServiceDateRequestBuilder.java

```diff
@@ -0,0 +1,66 @@
+package org.opentripplanner.transit.api.request;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.timetable.TripAlteration;
+
+public class TripOnServiceDateRequestBuilder {
+
+  private List<FeedScopedId> authorities = List.of();
+  private List<FeedScopedId> lines = List.of();
+  private List<FeedScopedId> serviceJourneys = List.of();
+  private List<FeedScopedId> replacementFor = List.of();
+  private List<String> privateCodes = List.of();
+  private List<TripAlteration> alterations = List.of();
+  private List<LocalDate> operatingDays;
```

Comment:
Now these can initially be `null`, because the null check is done in the constructor.
```suggestion
  private List<FeedScopedId> authorities;
  private List<FeedScopedId> lines;
  private List<FeedScopedId> serviceJourneys;
  private List<FeedScopedId> replacementFor;
  private List<String> privateCodes;
  private List<TripAlteration> alterations;
  private List<LocalDate> operatingDays;
```

Reply:
Implementing.